### PR TITLE
new math stuff

### PR DESCRIPTION
**Pull request by skonzy**
Some more math stuff

### CODE REVIEW

File: math_utils.py [hunk 1]
```diff
@@ -29,3 +29,70 @@ def factorial(n):
...
     return result
+
+def fibonacci(n):
+    """
+    Calculates the nth Fibonacci number.
+    """
+    if n < 0:
+        raise ValueError("Fibonacci is not defined for negative numbers")
+    if n == 0:
+        return 0
```
Comment (nate-prism) [inline math_utils.py:40]: Are you sure this works?

File: math_utils.py [hunk 2]
```diff
@@ -29,3 +29,70 @@ def factorial(n):
...
+    """
+    return sorted(s1) == sorted(s2)
+
+def is_armstrong(n):
+    """
+    Checks if a number is an Armstrong number.
+    """
+    s = str(n)
+    return n == sum(int(c) ** len(s) for c in s)
+
```
Comment (nate-prism) [inline math_utils.py:85]: I'm not sure this function will work. Have you tested it?

File: math_utils.py (hunk 1 above)
Comment (skonzy) [reply math_utils.py:40]: Yes, that works.

File: math_utils.py (hunk 2 above)
Comment (skonzy) [reply math_utils.py:85]: I'm sure it will work.

**Review by nate-prism — APPROVED** [review]
Sounds good!

**Pull request merged**